### PR TITLE
Feature/add git hooks

### PR DESCRIPTION
**Pull request by aehwany**

### CODE REVIEW

**Comment by aehwany** [comment]
@MohamedBeydoun dw will squash ;)

**Review by MohamedBeydoun — APPROVED** [review]
lgtm. I'd wait for approvals from @ValerianClerc and @idodin to make sure that we want to use husky.

**Review by imad-ant — APPROVED** [review]
lgtm - fine with husky, better than dealing with hooks manually

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "vue-cli-service lint --no-fix --max-warnings 0"
```
Comment (MohamedBeydoun) [inline package.json]: swap the names of these two commands (npm lint shouldnt fix while npm format will fix)

**Review by MohamedBeydoun — APPROVED** [review]

**Review by ValerianClerc — APPROVED** [review]
LGTM! Good find with husky, seems a lot nicer than manually writing pre-commit bash githooks. (don't forget to squash :P )

**Pull request merged**